src/commonMain/kotlin/fr/acinq/lightning/channel/states/WaitForInit.kt

```diff
@@ -58,10 +58,6 @@ object WaitForInit : ChannelState() {
                 val nextState = WaitForAcceptChannel(cmd, open)
                 Pair(nextState, listOf(ChannelAction.Message.Send(open)))
             }
-            cmd is ChannelCommand.Restore && cmd.state is Closing && cmd.state.commitments.nothingAtStake() -> {
-                logger.info { "we have nothing at stake, going straight to CLOSED" }
-                Pair(Closed(cmd.state), listOf())
-            }
```

Comment:
I'm not sure why we had this, we should have gone to Closed already. I suspect this comes from eclair, where there is a delay before cleaning up the channel.